autogen/cache/cache.py

```diff
@@ -1,8 +1,12 @@
+# Install Azure Cosmos DB SDK if not already
+
 from __future__ import annotations
 
 import sys
 from types import TracebackType
-from typing import Any, Dict, Optional, Type, Union
+from typing import Any, Dict, Optional, Type, TypedDict, Union
+
+from azure.cosmos import CosmosClient
```

Comment:
Should separate this into a different module to avoid import error when only `Cache` is needed.